Transferred the meta example entries from PR#32

sthagen opened the pull request:
## This PR replaces PR #72

The change-set transfers the missing changes from PR #72 into the already merged
variant (per PR #73)

### Changes:

- Added the meta example entries from PR#72
- Added Stefan Hagen as co-editor
- Validated file per markdownlint with configuration at 
  /contributions/dta/use-cases/etc/markdownlint.json

Co-authored-by: Lisa Bobbit

Propose to close the PR #72 without merge, when this change-set is merged into the main branch.

sthagen commented:
> Need to add Stefan as a participant in this document

@LisaBobbitt Done, already.

davaya reviewed (approved):
A Universally Unique Identifier (UUID) is a 128 bit value with the format defined in [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122), with a text format that looks like `e80adf88-b75c-4741-bc14-ce31f812bffd`.

The examples in, for example, 2.5.3 Unique Metadata Identifier, is `"UUID-1234-5678-9012-3456"`. If there is a fixed text tag in metadata identifiers, it should be anything but "UUID" unless the rest of the ID is an RFC-4122 value.  I'd recommend omitting the tag entirely, but if present, call it UID or metadata ID (MID) or anything else except UUID.

sthagen commented:
@davaya thank you for spotting the invalid UUIDs prefixed with "UUID-".

> A Universally Unique Identifier (UUID) is a 128 bit value with content defined in [RFC 4122](https://datatracker.ietf.org/doc/html/rfc4122), and a text representation that looks like `e80adf88-b75c-4741-bc14-ce31f812bffd`.
> 
> The examples in, for example, 2.5.3 Unique Metadata Identifier, is `"UUID-1234-5678-9012-3456"`. If there is a fixed text tag in metadata identifiers, it should be anything but "UUID" unless the rest of the ID is an RFC-4122 value. I'd recommend omitting the tag entirely, but if present, call it UID or metadata ID (MID) or anything else except UUID.

In commit [689f0ce](https://github.com/oasis-tcs/dps/pull/75/commits/689f0ceb7088f23e6c5fb428a69e5105c93b5a9f) I fixed these by replacing them with a random UUID4 value I created (preserving the intent of the surrounding sentence and making these metadata identifiers valid UUIDs.

In addition I completed the URLs to become valid that already carried valid UUIDs in another example.

@LisaBobbitt I amended our pull request, based on my assumption, that we fixed errors and can this way accelerated our progress. If I did assume wrongly, we can easily amend or revert. Thanks.
